api/registrations/serializers.py

```diff
@@ -95,6 +97,11 @@ def get_registered_meta(self, obj):
                 return meta_values
         return None
 
+    def get_embargo_end_date(self, obj):
+        if obj.embargo_end_date:
```

Comment:
if there is no embargo_end_date, False is returned.  False can't appear in a date-time field.